Samples/PhotoEditor/cs-winui/Package.appxmanifest

```diff
@@ -0,0 +1,48 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+
+<Package
+  xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10"
+  xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10"
+  xmlns:rescap="http://schemas.microsoft.com/appx/manifest/foundation/windows10/restrictedcapabilities"
+  IgnorableNamespaces="uap rescap">
+
+  <Identity
+    Name="8d1fcdb6-2142-4f1d-8a99-cf03513c7298"
+    Publisher="CN=bgursahani"
```

Comment:
change to Microsoft Corporation - see https://github.com/microsoft/WindowsAppSDK-Samples/blob/main/Samples/Windowing/cs-winui/Package.appxmanifest as an example